packages/block-library/src/template-part/edit/inner-blocks.js

```diff
@@ -36,7 +38,83 @@ function useLayout( layout ) {
 	}
 }
 
-export default function TemplatePartInnerBlocks( {
+const parsedBlocksCache = new WeakMap();
+
+function NonEditableTemplatePartPreview( {
+	postId: id,
+	layout,
+	tagName: TagName,
+	blockProps,
+} ) {
+	useBlockEditingMode( 'disabled' );
```

Comment:
Can we replace this with templateLock: 'all' ?

Reply:
I tried that first. But template lock all only prevents moving / deleting / inserting blocks. Not changing the attributes of any of the blocks that are already in there.

Reply:
I wish we could make another "lock" level rather than a hook like that but that's fine for now (declarative means less bugs in general)